Sources/Sentry/SentrySdkInfo.m

```diff
@@ -1,25 +1,25 @@
 #import "SentrySdkInfo.h"
 #import <Foundation/Foundation.h>
 
+#if SWIFT_PACKAGE
+static NSString *SENTRY_PACKAGE_INFO = @"spm:getsentry/%@";
+#elif COCOAPODS
+static NSString *SENTRY_PACKAGE_INFO = @"cocoapods:getsentry/%@";
+#elif CARTHAGE_YES
```

Comment:
Does `#if defined(CARTHAGE)` not work here? Why do we need to embed it into a new definition?

Reply:
Nope, it does not. 
CARTHAGE is a xcodebuild build setting, the only way I found to make it work is by "converting" it into a new definition.

Reply:
`m`: Please add the info you posted ⬆️ as a comment @brustolin.